### ISSUE TITLE
Error: Data types don't match while using more than 2946 rows newline delimited JSON data

### ISSUE THREAD

**Issue by epb984** [pola-rs/nodejs-polars#128]
#### Have you tried latest version of polars?

- yes

#### What version of polars are you using?

nodejs-polars 0.7.4

#### What operating system are you using polars on?

Microsoft Windows 11 Home v.10.0.22621 Build 22621

#### What node version are you using
node 16.18.0

#### Describe your bug.

Data types don't match error raised while using more than 2946 rows newline delimited JSON data.

#### What are the steps to reproduce the behavior?

```js
const { scanJson } = require("nodejs-polars")

const filename= "_hashtags-with-error.json"
const df = scanJson(filename, { inferSchemaLength: -1 }).collectSync()
```

Data I am using:
https://gist.github.com/epb984/d386ac436285b047ad5c7e25251c9b09

#### What is the actual behavior?

```
Error: Data types don't match: cannot vstack: because column datatypes (dtypes) in the two DataFrames do not match for left.name='hashtags' with left.dtype=list[str] != right.dtype=i32 with right.name='hashtags'
    at Object.collectSync (C:\rg-2023\node_modules\nodejs-polars\bin\lazy\dataframe.js:63:53)
    at Object.<anonymous> (C:\rg-2023\manualscan.js:9:58)
    at Module._compile (node:internal/modules/cjs/loader:1155:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1209:10)
    at Module.load (node:internal/modules/cjs/loader:1033:32)
    at Function.Module._load (node:internal/modules/cjs/loader:868:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:22:47 {
  code: 'GenericFailure'
}
```

#### What is the expected behavior?

execution finish with no error triggered

**Comment by universalmind303** [comment]
@epb984 have you tried with the latest version? [0.8.2](https://www.npmjs.com/package/nodejs-polars/v/0.8.2) ? There have been quite a few bugfixes upstream since 0.7.4

**Comment by universalmind303** [comment]
Confirmed this is an issue on the latest version & upstream polars.

**Comment by Bidek56** [comment]
@epb984  This seems to be fixed in `nodejs-polars==0.9.0`

**Comment by epb984** [comment]
thankyou @Bidek56
